modal_proto/api.proto

```diff
@@ -2769,6 +2769,7 @@ message TaskLogs {
   TaskProgress task_progress = 9;
   string function_call_id = 10;
   string input_id = 11;
+  string id = 12;
```

Comment:
what kind of id is this?

Reply:
It's a unique identifier but the value is something like `{task_id}-{timestamp_nanoseconds}`